site/content/docs/5.2/examples/form/index.html

```diff
@@ -0,0 +1,233 @@
+              </div>
+            </div>
+            <div class="col-12">
+              <div class="mb-3">
+                <label for="inputFirstName" id="inputFirstNameLabel" class="form-label is-required">First name</label>
+                <input type="text" class="form-control" id="inputFirstName"
+                  aria-labelledby="inputFirstNameLabel inputFirstNameFeedback" autocomplete="given-name" required>
+                <div class="valid-feedback">
+                  Looks good!
+                </div>
+                <div id="inputFirstNameFeedback" class="invalid-feedback">
+                  Please enter a valid first name.
+                </div>
+              </div>
+              <div class="mb-3">
+                <label for="inputLastName" id="inputLastNameLabel" class="form-label is-required">Last name</label>
+                <input type="text" class="form-control" id="inputLastName"
+                  aria-labelledby="inputLastNameLabel inputLastNameFeedback" autocomplete="family-name" required>
+                <div class="valid-feedback">
+                  Looks good!
+                </div>
+                <div id="inputLastNameFeedback" class="invalid-feedback">
+                  Please enter a valid last name.
+                </div>
+              </div>
+              <div class="mb-3">
+                <label for="inputEmail" id="inputEmailLabel" class="form-label is-required">Email</label>
+                <input type="email" class="form-control" id="inputEmail" aria-labelledby="inputEmailLabel inputEmailFeedback" autocomplete="email" required>
+                <div class="valid-feedback">
+                  Looks good!
+                </div>
+                <div id="inputEmailFeedback" class="invalid-feedback">
+                  Please provide a valid email.
+                </div>
+              </div>
+              <div class="mb-3">
+                <label for="selectCountry" id="selectCountryLabel" class="form-label is-required">Country</label>
+                <select class="form-select" id="selectCountry"
+                  aria-labelledby="selectCountryLabel selectCountryFeedback" autocomplete="country-name" required>
+                  <option selected disabled value="" aria-hidden="true">Select a country</option>
+                  <option value="1">France</option>
+                  <option value="2">Canada</option>
+                  <option value="3">USA</option>
+                  <option value="4">Australia</option>
+                </select>
+                <div class="valid-feedback">
+                  Looks good!
+                </div>
+                <div id="selectCountryFeedback" class="invalid-feedback">
+                  Please select a valid country.
+                </div>
+              </div>
+              <div class="mb-3">
+                <label for="inputCompany" id="inputCompanyLabel" class="form-label is-required">Name of
```

Comment:
There's a formatting issue in the file. Based on Bootstrap, for one HTML tag we don't use break lines. In this case, everything must be on the same line.